mobly/records.py

```diff
@@ -729,19 +729,34 @@ def summary_dict(self):
 
 
 class SuiteInfoRecord:
-  """A record representing the suite info in test summary."""
+  """A record representing the suite info in test summary.
 
-  KEY_SUITE_CLASS_NAME = 'Suite Class Name'
-  KEY_TIMESTAMP = 'Timestamp'
+  Attributes:
+    suite_class: str, the class name of the test suite class.
+    begin_time: int, epoch timestamp of when the suite started.
+    end_time: int, epoch timestamp of when the suite ended.
+  """
 
-  def __init__(self, suite_class_name):
-    self.suite_class_name = suite_class_name
-    self.timestamp = time.time()
+  KEY_SUITE_CLASS = 'Suite Class'
```

Comment:
what's "suite class"?

Reply:
The test suite class that inherits from `base_suite.BaseSuite`. 

If "Suite Class" is not clear enough, I modified it to "Test Suite Class".